docs/GETTING_STARTED_BEGINNER.md

```
@@ -0,0 +1,1109 @@
+Write-Host "✅ Created test file: $testFile"
+```
+
+### Sync File to Neo4j
+
+```bash
+# Extract frontmatter and create Neo4j node
+poetry run python << 'EOF'
+import frontmatter
+from pathlib import Path
+from neo4j import GraphDatabase
+from omega_kg.settings import settings
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)